samples/rag-aisearch/README.md

```diff
@@ -73,3 +73,24 @@ Semantic Search binding for Azure AI Search expects/creates a very specific inde
 ### Vector Search Configuration
 
 Algorithm - HnswAlgorithmConfiguration
+
+### Use of API Key
+
```

Comment:
I think this section needs an introduction explaining what API keys are for, that they're optional, and that managed identities are recommended.